$NetBSD: patch-ab,v 1.2 2012/06/10 16:40:24 wiz Exp $

Ugh ... more arbitrary per-OS preprocessor fun; let's replace this
with something we can keep track of more intuitively.

--- sessreg.h.orig	2011-06-26 16:30:31.000000000 +0000
+++ sessreg.h
@@ -76,7 +76,7 @@
 #endif
 
 #ifdef CSRG_BASED
-#if !defined(__FreeBSD__) && !defined(__OpenBSD__)
+#ifdef NO_COLON_ZERO_IN_UTMP
 /* *BSD doesn't like a ':0' type entry in utmp */
 #define NO_UTMP
 #endif
